.github/workflows/zig-fmt.yml

```diff
@@ -0,0 +1,17 @@
+name: Zig fmt
+on: [push, pull_request, workflow_dispatch]
```

Comment:
Did you purposefully not specify the branches for the `push` and `pull_request` events? This will result in two runs for pushes to a pull request.